tests/func/test_get.py

```diff
@@ -174,7 +174,12 @@ def test_get_from_non_dvc_master(tmp_dir, erepo_dir, caplog):
 
     caplog.clear()
     dst = "file_imported"
-    with caplog.at_level(logging.INFO, logger="dvc"):
+    with (
+        caplog.at_level(logging.INFO, logger="dvc"),
+        # removing `git` import in conftest resulted in unexpected logs from
+        # that package, see https://github.com/iterative/dvc/issues/3167
+        caplog.at_level(logging.INFO, logger="git"),
+    ):
```

Comment:
parentheses do not seem to be supported
ref: https://stackoverflow.com/a/50958431